### ISSUE TITLE
An Invalid Peer was Used

### ISSUE THREAD

**Issue by ghost** [alemonmk/pytgasu#2]
Any idea what is `Peer` in this case?

My assumption is something is wrong here, https://github.com/alemonmk/pytgasu/blob/2740a7221599ae6ecea4139b6624a31876b942e6/pytgasu/upload/uploader.py#L26

```
(venv) ➜  pytgasu git:(master) pytgasu upload ../sticker
Preparing ../sticker for upload...
Traceback (most recent call last):
  File "/Users/xli/Desktop/pytgasu/venv/bin/pytgasu", line 11, in <module>
    sys.exit(cli())
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/pytgasu/cli.py", line 73, in upload
    SetUploader(tc=tc, sets=sticker_sets, subscribe=s)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/pytgasu/upload/uploader.py", line 44, in upload
    send_bot_cmd(msg=['/cancel', '/start'])
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/pytgasu/upload/uploader.py", line 94, in _send_bot_cmd
    res = tc.send_message(entity=_stickersbot, message=m)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/telethon/telegram_client.py", line 788, in send_message
    result = self(request)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/telethon/telegram_bare_client.py", line 450, in __call__
    result = self._invoke(call_receive, *requests)
  File "/Users/xli/Desktop/pytgasu/venv/lib/python3.6/site-packages/telethon/telegram_bare_client.py", line 542, in _invoke
    raise next(x.rpc_error for x in requests if x.rpc_error)
telethon.errors.rpc_error_list.PeerIdInvalidError: An invalid Peer was used. Make sure to pass the right peer type
```

**Comment by alemonmk** [comment]
Sorry for being so late on looking into this issue.

I literally copy and paste this InputPeerUser into my Python interpreter session and the send_message() works. Please try it yourself so we can get to the root of it.

```
>>> from telethon.tl.types import InputPeerUser
>>> p = InputPeerUser(user_id=429000, access_hash=9143715803499997149)
>>> tc.send_message(entity=p, message='/cancel')
<telethon.tl.types.Message object at 0x00000063800C4320>
>>>
```

**Comment by alemonmk** [comment]
Should be fixed by commit e46fb0a.
